sysconfig/cloudinit.go

```diff
@@ -124,6 +125,10 @@ func configureCloudInit(opts *Options) (err error) {
 		// example on test devices where the gadget has a gadget.yaml, but for
 		// testing purposes you also want to provision another user with
 		// ubuntu-seed cloud-init config
+
+		// set the prefix such that any ubuntu-seed config that ends up getting
+		// installed takes precedence over the gadget config
+		targetFilesPrefix = "90_"
```

Comment:
I think things will be easier to reason about if we just apply this prefix unconditionally to the config from ubuntu-seed

Reply:
sure, now they always have that prefix